lib/TokenManager.js

```diff
@@ -24,51 +24,50 @@ function emitExpired(tokenMgmtRef, key, token) {
   tokenMgmtRef.emitter.emit('expired', key, token);
 }
 
-function clearRefreshTimeout(tokenMgmtRef, key) {
-  clearTimeout(tokenMgmtRef.refreshTimeouts[key]);
-  delete tokenMgmtRef.refreshTimeouts[key];
+function emitError(tokenMgmtRef, error) {
+  tokenMgmtRef.emitter.emit('error', error);
 }
 
-function clearRefreshTimeoutAll(tokenMgmtRef) {
-  var refreshTimeouts = tokenMgmtRef.refreshTimeouts;
-  for(var key in refreshTimeouts) {
-    if (!refreshTimeouts.hasOwnProperty(key)) {
+function clearExpireEventTimeout(tokenMgmtRef, key) {
+  clearTimeout(tokenMgmtRef.expireTimeouts[key]);
+  delete tokenMgmtRef.expireTimeouts[key];
+}
+
+function clearExpireEventTimeoutAll(tokenMgmtRef) {
+  var expireTimeouts = tokenMgmtRef.expireTimeouts;
+  for(var key in expireTimeouts) {
+    if (!expireTimeouts.hasOwnProperty(key)) {
       continue;
     }
-    clearRefreshTimeout(tokenMgmtRef, key);
+    clearExpireEventTimeout(tokenMgmtRef, key);
   }
-  tokenMgmtRef.refreshTimeouts = {};
+  tokenMgmtRef.expireTimeouts = {};
```

Comment:
We could just do `delete tokenMgmtRef.expireTimeouts[key];` inside the for loop, instead of creating a new object?

Reply:
Yep - we already do: [TokenManager.js#L33](https://github.com/okta/okta-auth-js/blob/751733e36ae2efcc4388450e53000139396690fa/lib/TokenManager.js#L33). I'm assuming this logic is a failsafe.

Reply:
Ah, so then why is there a need to reassign a new object?

Reply:
Legacy code - it has now been removed.